infra/filter.lua

```diff
@@ -1,5 +1,7 @@
 -- Pass 1: Load configuration data
 
+local json = require('infra/dkjson')
```

Comment:
There is already a JSON file (config.json) which is somehow parsed by pandoc and passed to this filter. Can we extend it instead of importing a second JSON parser? Another option is to pass additional yaml files on the commandline (see section 8.10.2 [here](https://pandoc.org/chunkedhtml-demo/8.10-metadata-blocks.html)).

Reply:
Thanks you your comment, I discovered that `pandoc.json.encode` exists, so I was able to use that to encode the JSON representation of the quiz. It's not called explicitly anywhere in the project, (pandoc calls it implicitly during the build process) which might explain why I didn't find it the first time. 😅 Using the JSON encoder provided by pandoc is a much better solution; thank you for pushing back on this!